.github/workflows/docs-release.yml

```
@@ -0,0 +1,57 @@
+name: Docs Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch: # Enables manual triggering of the workflow
+
+jobs:
+  build:
+    name: Build Docusaurus
+    runs-on: ubuntu-latest
```

We need to use a different runner

```suggestion
    runs-on: 
        group: databrickslabs-protected-runner-group
        labels: linux-ubuntu-latest
```

---

Done, fixed